scripts/test_generate.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+SCRIPTS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+CONTROLLER_BOOTSTRAP_DIR="$SCRIPTS_DIR/.."
+CONTROLLER_BOOTSTRAP="$CONTROLLER_BOOTSTRAP_DIR/bin/controller-bootstrap"
+CONTROLLER_BOOTSTRAP_TEMPLATES_DIR="$CONTROLLER_BOOTSTRAP_DIR/templates"
+SERVICE="eks"
+CONTROLLER_NAME="$SERVICE-controller"
+CONTROLLER_DIR="$CONTROLLER_BOOTSTRAP_DIR/test_output/$CONTROLLER_NAME"
+
+export SERVICE
+export CONTROLLER_DIR
+cd $CONTROLLER_BOOTSTRAP_DIR
+make generate
+
+if [[ ! -e "$CONTROLLER_DIR/TEST_FILE" ]]; then
+  echo "'TEST_FILE' is not found in $CONTROLLER_NAME. Creating a new template test file, controller-bootstrap/templates/TEST_FILE.tpl ..."
```

Comment:
nit: for this echo and other echo statements too. This helps with debugging and gaining more context.

```suggestion
  echo "test_generate.sh][INFO] 'TEST_FILE' is not found in $CONTROLLER_NAME. Creating a new template test file, controller-bootstrap/templates/TEST_FILE.tpl ..."
```